after-install.sh

```diff
@@ -9,15 +10,17 @@ create_user() {
   FILES="${FILES} /usr/local/bin/lua2json"
   FILES="${FILES} /usr/local/bin/luarocks"
   FILES="${FILES} /usr/local/bin/luarocks-admin"
+  FILES="${FILES} /usr/local/bin/openapi2kong"
   FILES="${FILES} /usr/local/etc/luarocks/"
+  FILES="${FILES} /usr/local/etc/passwdqc/"
   FILES="${FILES} /usr/local/kong/"
   FILES="${FILES} /usr/local/lib/lua/"
   FILES="${FILES} /usr/local/lib/luarocks/"
   FILES="${FILES} /usr/local/openresty/"
   FILES="${FILES} /usr/local/share/lua/"
 
   for FILE in ${FILES}; do
-    chown -R kong:kong ${FILE}
+    chown -R kong:root ${FILE}
     chmod -R g=u ${FILE}
```

Comment:
Implementing the control described in the comment at https://github.com/Kong/kong-build-tools/pull/457#issuecomment-1125126539, the chmod should really be `chmod -R g=u,g-w` unless the group actually needs write access.

Related: why is this in a `for` loop?  Just run like `chown $FILES` and let the shell's whitespace splitting work it out; that's already what's happening in the for loop anyway, and there's no sense fork/exec'ing chown & chmod 15 times. ;)

Reply:
One point is that we shouldn't be making new folders at the filesystem root to begin with.  But perhaps that's a separate problem. :D

Reply:
Option 2 is the higher security approach.  Using root:root with other access gives access to all processes on the entire machine.  Using root:kong with the world access bits removed limits access to only members of the kong group.

The reason nginx provides world access to its config is that nginx is a general-purpose web server.  Its use cases include serving user home directories and being run as a proxy by other processes on the system.  So it is a service which may need to be run by any user on the system while including its global config.  Kong gateway is a different use case - it is intended to be run by only the Kong administrator, and then any non-administrative users who interact with it are expected to only use the API.  So, the only users who need to access its files directly are Kong administrators and the process itself.

Reply:
FWIW: I checked the Fedora, Debian, arch, and SUSE (ok, I was a SUSE release coordinator, so I actually just remembered that one) packaging guides earlier today.  They all say root:root/0755 for binaries unless increased security is necessary (a suid or sgid binary, for example).  None have a specific guideline for configuration, so I think we're generally free to do what makes most sense to us.

Reply:
> One point is that we shouldn't be making new folders at the filesystem root to begin with. But perhaps that's a separate problem. :D

Maybe, we should let Kong creates runtime data folder dynamically under `/run/` or `/var/run`. It's the de facto. Agree that, this is another topic.

Reply:
> FWIW: I checked the Fedora, Debian, arch, and SUSE (ok, I was a SUSE release coordinator, so I actually just remembered that one) packaging guides earlier today. They all say root:root/0755 for binaries unless increased security is necessary (a suid or sgid binary, for example). None have a specific guideline for configuration, so I think we're generally free to do what makes most sense to us.

Ha, really appreciate the time to do the research. Leaving it as default (`root:root`) is preferable, as that will make our Kong package as much compatible with different Linux distributions as possible.

I will file a commit to this PR.